lib/dash/content_protection.js

```diff
@@ -276,7 +449,19 @@ shaka.dash.ContentProtection.convertElements_ = function(
           goog.asserts.assert(!element.init || element.init.length,
                               'Init data must be null or non-empty.');
           var initData = element.init || defaultInit;
-          return [ManifestParserUtils.createDrmInfo(keySystem, initData)];
+          var drmInfo = ManifestParserUtils.createDrmInfo(keySystem, initData);
+
+          // extract Widevine license URL
+          if (keySystem === 'com.widevine.alpha') {
```

Comment:
To make it easier for us to add support for other schemes, could we structure this like:

```
var license_url_parsers = {
  'com.widevine.alpha': shaka.dash...
  'com.microsoft.playready': shaka.dash...
};

var license_parser = license_parsers[keySystem];

if (license_parser) {
  drmInfo.licenseServerUri = license_parser(element);
}
```